Godot_test/ClientScripts/UIButtons.cs

```diff
@@ -0,0 +1,40 @@
+using Godot;
+using System;
+
+public partial class UIButtons : Control
+{
+  private void _on_stats_button_pressed()
+  {
+	var playerStatsMenu = (PlayerStatsMenu)GetParent().GetNode("PlayerStatsMenu");
+	if (!playerStatsMenu.isOn)
```

Comment:
It would be better to have one method with a content of that if, and call it also in else. Method would get parameters which differ case if from lese.